h/static/scripts/annotator/plugin/test/pdf-metadata-test.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+var PDFMetadataService = require('../pdf-metadata');
+
+describe('pdf-metadata', function () {
+  var pdfMetadata;
+  var fakePDFViewerApplication = {
+    'documentFingerprint': 'fakeFingerprint',
+    'documentInfo': {
+      'Title': 'fakeTitle',
+    },
+    'metadata': {
+      'metadata': {
+        'dc:title': 'fakeTitle',
+      }
+    },
+    'url': 'fakeUrl',
+  };
+  var sandbox;
```

Comment:
TBH using a sandbox is good practice if you have state that lives outside a test method, as it makes it much harder to forget to tear it down.